compiler+runtime/src/cpp/jank/runtime/obj/inst.cpp

```diff
@@ -0,0 +1,102 @@
+#include <chrono>
```

Comment:
No need to include things which are already in the `inst.hpp` header which we're including.